swaggerpy/client.py

```diff
@@ -178,7 +178,7 @@ def _construct_request(self, **kwargs):
         _request_options = kwargs.pop('_request_options', {}) or {}
 
         request = {}
-        request['method'] = self._json[u'method']
+        request['method'] = str(self._json[u'method'])
```

Comment:
Hey @dnephin - for my own edification, what's the reasoning behind that? Just that str() uses the default encoding, which can vary between environments? If so, we should probably change the async client since it also casts unicode fields to strings with str().